mmdet/core/nncf/compression_hooks.py

```diff
@@ -0,0 +1,26 @@
+from texttable import Texttable
```

Comment:
add to requirements.txt

Reply:
I'd prefer to make this import happen only if it's actually required, i.e. if nncf is installed and used.

Also it's good to add nncf itself to the list of optional dependencies.

Reply:
Done